gensrc/thrift/FrontendService.thrift

```diff
@@ -45,6 +45,9 @@ struct TSetSessionParams {
 struct TAuthenticateParams {
     1: required string user
     2: required string passwd
+    3: required string host
+    4: required string db_name
+    5: required list<string> table_names;
```

Comment:
I suggest you add privilege to the definition of thrift for further usage.

Reply:
As discussed offline, we will add privilege on demand in the future.